Installation instructions

kristinawlai opened the pull request:
Review updates to installation instructions

d-morrison reviewed (changes requested):
This branch shouldn't include the new vignette ('typhoid-seroincidence.Rmd'); let's remove it and see if that resolves the check errors?

d-morrison committed:
Delete README.html

This file shouldn't be tracked in git.

kristinawlai committed:
Merge pull request #1 from UCD-SERG/pr8-patch-1

Pr8 patch 1

d-morrison commented on vignettes/installation.Rmd:
```diff
@@ -83,19 +83,19 @@ Graphical user interface for R interpreter will start in a new window:
 use. The package can be installed either through a menu option in the graphical user interface (GUI) 
 or with a command executed in the R console. Both options are presented:
 
 ### 2.2.1. Package installation from the graphical user interface
```
This subsection is out of date; remove?

kristinawlai replied:
Replaced with updated installation instructions (or so I think I did...)

d-morrison reviewed (changes requested):
Looks good, except for subsection about installing from downloaded binary files, which I don't think is currently an available option. Let's remove it?

kristinawlai committed:
Installation updates

Updated installation instructions in README and vignette to reflect development version. Added .html files to .gitignore.

kristinawlai committed:
Merge branch 'installation_instructions' of https://github.com/kristinawlai/serocalculator into installation_instructions

kristinawlai committed:
Merge branch 'UCD-SERG:main' into installation_instructions

d-morrison committed:
we don't want to actually run `install.packages()` in our vignette, just display the code. We can do that like this, or with the chunk option "eval = FALSE".

kristinawlai committed:
Update installation.Rmd

kristinawlai committed:
Merge pull request #2 from UCD-SERG/PR8-patch-2

fix installation.Rmd

d-morrison reviewed (approved):
Looks good!
